tests/unit/components/DocumentationTopic/Summary/Availability.spec.js

```diff
@@ -74,52 +75,45 @@ describe('Availability', () => {
     expect(section.attributes('role')).toBe('complementary');
   });
 
-  it('renders a `Badge` for technologies, a `Badge` and `AvailabilityRange` for each platform', () => {
+  it('renders a `div` for technologies, a `AvailabilityRange` and a `Badge` if applicable for each platform', () => {
```

Comment:
```suggestion
  it('renders a `span` for technologies, a `AvailabilityRange` and a `Badge` if applicable for each platform', () => {
```

(or maybe update the whole test name to reflect how it works today in case it's different)